UploadDaemon/UploadDaemon.cs

```diff
@@ -18,7 +18,7 @@ namespace UploadDaemon
     /// </summary>
     public class UploadDaemon
     {
-        private const long TimerIntervalInMilliseconds = 1000 * 60 * 5;
+        private const long MinuteToMillisecondsFactor = 1000 * 60;
```

Comment:
use TimeSpan.FromMinutes(minutes).TotalMilliseconds instead?

Reply:
Yes! Didn't know this exists 😇